### PR TITLE
Fix nav

### PR DESCRIPTION
**Pull request by rinto-c**

### CODE REVIEW

**Comment by rinto-c** [comment]
@jesseyuen - please review

File: app/assets/sass/components/_eyfs-mobile-navigation.scss [hunk 1]
```diff
@@ -7,10 +7,7 @@
   .app-mobile-nav {
```
Comment (jesseyuen) [inline app/assets/sass/components/_eyfs-mobile-navigation.scss:7]: You have styles for this class spread across both `_eyfs-mobile-navigation.scss` and `_sticky-nav--test.scss`. Consolidate the styles under one selector and keep the styles in only one sass partial.

File: app/assets/sass/components/_sticky-nav--test.scss [hunk 2]
```diff
@@ -1,89 +1,90 @@
...
-          @include govuk-shape-arrow($direction: up, $base: 10px, $display: inline-block);
-          border-bottom-color: currentColor;
-        }
-    
-        .js-enabled & {
-          @include govuk-media-query($until: tablet) {
-            display: inline;
-          }
-        }
+    &,
```
Comment (jesseyuen) [inline app/assets/sass/components/_sticky-nav--test.scss:40]: What is this lone `&,` doing?

File: app/assets/sass/components/_sticky-nav--test.scss [hunk 3]
```diff
@@ -1,89 +1,90 @@
...
+    padding: govuk-spacing(3) govuk-spacing(3);
+    margin: 0;
+    border-bottom: 1px solid govuk-tint(govuk-colour("mid-grey"), 50);
+    // border: 1px solid transparent; // Override the button default border, keep this around for when users change their colours
+    box-shadow: none; // Override the button default box shadow
+    color: $govuk-link-colour;
+    font-size: 1.2rem;
+    font-weight: 700;
+    text-align: left;
+    &:hover {
```
Comment (jesseyuen) [inline app/assets/sass/components/_sticky-nav--test.scss:36]: You have declared two sets of `&:hover` rules under the same selector. Consolidate these.

File: app/views/includes/sticky-nav--test.html [hunk 4]
```diff
@@ -1,8 +1,8 @@
-<div class="govuk-grid-column-full">
+{# <div class="govuk-grid-column-full"> #}
```
Comment (jesseyuen) [inline app/views/includes/sticky-nav--test.html:1]: Delete this commented out markup

File: app/views/includes/sticky-nav--test.html [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
     <div id="eyfs-sticky-nav__mobile" class="eyfs-sticky-nav eyfs-sticky-nav--clickable">
         <button id="eyfs-sticky-nav-toggler" class="govuk-button eyfs-sticky-nav__button eyfs-sticky-nav__link eyfs-sticky-nav-toggler js-eyfs-sticky-nav-toggler" aria-controls="app-mobile-nav">
```
Comment (jesseyuen) [inline app/views/includes/sticky-nav--test.html:3]: Im wondering why this button has so many classes `govuk-button eyfs-sticky-nav__button eyfs-sticky-nav__link eyfs-sticky-nav-toggler js-eyfs-sticky-nav-toggler`. 

Are all of them necessary? Are you spreading style/function across multiple classes? I imagine you could simplify this. The class names are also all quite similar so dont necessarily give you an idea of the how/why the purpose of each class just from reading it.

Comment (jesseyuen) [inline app/views/includes/sticky-nav--test.html:3]: `aria-controls="app-mobile-nav"` is a way to tell screen readers that this button controls a nav with an id of `app-mobile-nav`. Your nav is missing the id.

File: app/assets/sass/components/_sticky-nav--test.scss [hunk 6]
```diff
@@ -1,89 +1,90 @@
...
-                }
+            &:focus {
+                color: $govuk-focus-colour;
             }
         }
     }
+}
+.eyfs-sticky-nav-toggler {
+    // display: none;
+    padding: govuk-spacing(3) govuk-spacing(3);
```
Comment (jesseyuen) [inline app/assets/sass/components/_sticky-nav--test.scss:27]: You only need to declare `govuk-spacing(3)` once here

File: app/assets/sass/components/_sticky-nav--test.scss [hunk 7]
```diff
@@ -1,89 +1,90 @@
 .eyfs-sticky-nav {
     background-color: govuk-colour("white");
-    border-bottom: 1px solid govuk-tint(govuk-colour("mid-grey"), 50);
     position: relative;
```
Comment (jesseyuen) [inline app/assets/sass/components/_sticky-nav--test.scss:3]: As a general rule of best practice consider grouping your css rules by type: https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

...and in a specific order: https://9elements.com/css-rule-order/

File: app/assets/sass/components/_sticky-nav--test.scss (hunk 2 above)
Comment (rinto-c) [inline app/assets/sass/components/_sticky-nav--test.scss:40]: it is not in the actual code - weird! @jesseyuen

**Pull request closed without merging**